core/roboconf-agent/src/main/java/net/roboconf/agent/internal/Agent.java

```diff
@@ -82,7 +81,7 @@
 	 */
 	public Agent() {
 		this.logger = Logger.getLogger( getClass().getName());
-		this.messagingFactoryType = MessagingConstants.FACTORY_RABBIT_MQ;
+		this.messagingType = MessagingConstants.FACTORY_RABBIT_MQ;
```

Comment:
Well, that was in order to stay compatible with existing code.
When working w/ iPOJO, a default value can be injected, or we can put it in the configuration.
Without iPOJO (e.g. tests), the messaging type must be set explicitly. While I made my best to call `switchMessagingType(String)` in all the tests I've touched, I'm pretty sure I forgot some.

So I will remove this line as soon as the other issues are fixed!